src/db.ts

```diff
@@ -28,42 +51,76 @@ interface InternalReminder {
 	ping: string | null;
 }
 
-// -- Activity Tracker Database --
-export const activityTrackerBlacklistDb = new Enmap<string[]>("trackerBlacklist");
-activityTrackerBlacklistDb.ensure("general-user", []);
-activityTrackerBlacklistDb.ensure("general-game", []);
-
-// key: "[user id]-[game]"
-export const activityTrackerLogDb = new Enmap<ActivityLogEntry[], InternalActivityLogEntry[]>({
-	name: "trackerLog",
-	serializer: (data) => data.map((e) => ({ duration: e.duration.toISO(), date: e.date.toISO() })),
-	deserializer: (data) =>
-		data.map((e) => ({
-			duration: Duration.fromISO(e.duration),
-			date: DateTime.fromISO(e.date, { setZone: true }),
-		})),
-});
+/*
+-----------------------------
+- Activity Tracker Database -
```

Comment:
Please be consistent about the naming. Either activity tracker or just tracker. (This line is just a representative for the other times you used the "activity"

Reply:
I think I added this before I deleted everything